File: Java_TP2/Codingan/Aksesories.java
public class Aksesories extends Petshop {
    private String Size;
    private String Material;
    private String Brand;

    public Aksesories() {
        super();
        this.Size = "";
        this.Material = "";
        this.Brand = "";
    }

    public Aksesories(int ID, String Name, String Category, int Price, int Stock, String Size, String Material, String Brand) {
        super(ID, Name, Category, Price, Stock);
        this.Size = Size;
        this.Material = Material;
        this.Brand = Brand;
    }

    public void settingDataAksesories(int ID, String Name, String Category, int Price, int Stock, String NewSize, String NewMaterial, String NewBrand) {
        super.settingDataPetshop(ID, Name, Category, Price, Stock);
        this.Size = NewSize;
        this.Material = NewMaterial;
        this.Brand = NewBrand;
    }

    public String getterSize() {
        return this.Size;
    }

    public String getterMaterial() {
        return this.Material;
    }

    public String getterBrand() {
        return this.Brand;
    }
}

File: Python_TP2/Codingan/Aksesories.py
from Petshop import Petshop

class Aksesories(Petshop):
    def __init__(self, ID, Name, Category, Price, Stock, Size="", Material="", Brand=""):
        super().__init__(ID, Name, Category, Price, Stock)
        self.Size = Size
        self.Material = Material
        self.Brand = Brand

    def setting_data_aksesories(self, ID, Name, Category, Price, Stock, NewSize, NewMaterial, NewBrand):
        super().setting_data_petshop(ID, Name, Category, Price, Stock)
        self.Size = NewSize
        self.Material = NewMaterial
        self.Brand = NewBrand

    def getter_size(self):
        return self.Size

    def getter_material(self):
        return self.Material

    def getter_brand(self):
        return self.Brand